### ISSUE TITLE
yolo_cpp_dll.dll returns bad data in debug

### ISSUE THREAD

**Issue by iand111** [AlexeyAB/darknet#3889]
I need a debug version of yolo_cpp_dll.dll. Ultimately my goal is to get a debug and release dll with cudnn and OpenCV enabled. I will enable these two once I can use the stock dll in debug. 

The debug dll builds successfully, but returns bad detections when I call it in my application. The release dll returns correct detections with the same config and image files. The only change I have made to the yolo_cpp_dll.vcxproj, after unzipping darknet, is adding a sub-directory to the outdir variable to differentiate between the two builds.

In the image below you can see the first detection appears to be correct and later detections are error values. 
![Dll Error](https://user-images.githubusercontent.com/6231804/64543586-560e6c00-d2da-11e9-9328-52b5e6325fb4.PNG)

Any suggestions would be greatly appreciated.

**Comment by AlexeyAB** [comment]
Did you compile both DLL and EXE in debug mode?

**Comment by iand111** [comment]
I am assuming you mean my application's .exe, not darknet.exe. If so, yes, both are compiled in debug.  

I have a older version of the darknet repo (possibly a commit from 2/18/2019?) that will compile and give me expected detection with yolo_cpp_dll.dll in debug. Currently trying to get this version to use opencv 3.4.7

**Comment by AlexeyAB** [comment]
@iand111 So old version 2/18/2019 works perfectly in Debug mode, but current version doesnt?

**Comment by iand111** [comment]
That is correct. Based on the readme the latest commit I could have pulled is 2/21.

**Comment by iand111** [comment]
@AlexeyAB My apologize, I had the old dll loaded in release. With the new dll, I get a similar result in release. The old version release dll returns correct detections, but the current version returns the same detections shown in the picture above.
